frontend/test/services/transaction.test.js

```diff
@@ -41,107 +36,18 @@ describe('Transaction Functions', () => {
     };
 
     connect.mockResolvedValue(mockStarknet);
-
-    process.env.REACT_APP_BACKEND_URL = mockBackendUrl;
-  });
-
-  afterEach(() => {
-    delete process.env.REACT_APP_BACKEND_URL;
-  });
-
-  describe('sendTransaction', () => {
```

Comment:
you must not deleted tests. it should be the same amount as we have in main now, I see 8 less

Reply:
unresolved.